fusedrug/data/tokenizer/modulartokenizer/pretrained_tokenizers/bmfm_extended_modular_tokenizer/bpe_tokenizer_trained_on_chembl_zinc_with_aug_4272372_samples_balanced_1_1.json

```diff
@@ -2842,6 +2869,7 @@
     "end_of_word_suffix": null,
     "fuse_unk": false,
     "byte_fallback": false,
+    "ignore_merges": false,
```

Comment:
what is this?

Reply:
After we add tokens to fuse-drug special_tokens.py, we need to run the script update_special_tokens_for_all_modular_tokenizers.sh, and that script automatically added that change. 
I talked with Vadim and he said it's OK. The false value is the default value, so there is no change in functionality in comparison to previous version.